scss/sideBar.module.scss

```diff
@@ -0,0 +1,144 @@
+$primary-color: hsl(0, 0%, 98.4%);
+
+.sidebar_title {
+  width: 327px;
+  height: 22px;
+  left: 0px;
+  top: 0px;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  text-align: center;
+  color: #000000;
+  margin-bottom: 22px;
+}
+
+.sidebar_mainbody {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 28px 21px;
+
+  position: static;
+  width: 369.01px;
+  height: 460px;
+  left: 0px;
+  top: 0px;
+
+  background: $primary-color;
+  border-radius: 10px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0px 39px;
+}
+
+.sidebar_tabsNav {
+  transition: color 0.25s ease-in-out, background-color 0.15s ease-in-out,
+    border-color 0.25s ease-in-out;
+  position: static;
+  width: 327.01px;
+  height: 52px;
+  background: $primary-color;
+  border-radius: 10px;
+  padding-left: 6.12%;
+  display: flex;
+  justify-content: space-between;
+  text-decoration: none;
+  font-family: 'Inter';
```

Comment:
We don't have Inter yet. Please create a PR that [adds](https://nextjs.org/docs/basic-features/font-optimization#usage) the font Inter before merging this PR.